### PR TITLE
New page to view domain import events

### PR DESCRIPTION
**Pull request by gmethvin**
Pretty basic right now. Just shows date, event type and a description for all events.

### CODE REVIEW

File: shoebox/app/com/keepit/controllers/admin/SliderAdminController.scala [hunk 1]
```diff
@@ -149,4 +154,36 @@ object SliderAdminController extends FortyTwoController {
     inject[DomainTagImporter].refetchClassifications()
     Ok(JsObject(Seq()))
   }
+
+  def getImportEvents = AdminHtmlAction { implicit request =>
```
Comment (eishay) [inline shoebox/app/com/keepit/controllers/admin/SliderAdminController.scala:158]: We're using the same strings around the codebase (as in DomainTagImportActor and here). Lets have them as constants so the compiler will help us a bit with typos.

File: shoebox/app/views/domainImportEvents.scala.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@(events: Seq[com.keepit.controllers.admin.ImportEvent])
+
+@admin("Domain Import Events", stylesheets = List("admin_slider")) {
+<table class="table table-condensed table-striped">
```
Comment (eishay) [inline shoebox/app/views/domainImportEvents.scala.html:4]: nice and simple, like it

File: shoebox/app/com/keepit/controllers/admin/SliderAdminController.scala [hunk 3]
```diff
@@ -149,4 +155,36 @@ object SliderAdminController extends FortyTwoController {
...
+    val selector = MongoSelector(EventFamilies.DOMAIN_TAG_IMPORT)
+    val dbObjects = inject[MongoEventStore].find("server", selector).toList
+    val failureSelector = MongoSelector(EventFamilies.EXCEPTION).withEventName(IMPORT_FAILURE)
+    val failureDbObjects = inject[MongoEventStore].find("server", failureSelector).toList
+    val events = (dbObjects ++ failureDbObjects).map { obj =>
```
Comment (andrewconner) [inline shoebox/app/com/keepit/controllers/admin/SliderAdminController.scala]: Glad that worked.

**Comment by gmethvin** [comment]
Any more comments?

File: shoebox/app/com/keepit/classify/DomainTagImporter.scala [hunk 4]
```diff
@@ -32,11 +32,27 @@ private case object RefetchAll
 private case class ApplyTag(tagName: DomainTagName, domainNames: Seq[String])
 private case class RemoveTag(tagName: DomainTagName)
 
+object DomainTagImportEvents {
+  // success
```
Comment (andrewconner) [inline shoebox/app/com/keepit/classify/DomainTagImporter.scala:36]: Might we want type safety here?

**Comment by 2is10** [comment]
None from me.

Doh!

**Comment by andrewconner** [comment]
Looks good to me.

**Pull request merged**